Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN pip3 install --no-cache-dir colorlog cython
 
 # For the nmap tracker, bluetooth tracker, Z-Wave
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends nmap net-tools cython3 libudev-dev sudo libglib2.0-dev bluetooth libbluetooth-dev && \
```

Comment:
The Docker image is not running on a Raspberry Pi as it only gets built for x86 platforms. So no need to add it.